<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCapturePrefWidget</class>
 <widget class="QWidget" name="VideoCapturePrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频捕获</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>截图模板名称:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="mChooseDir">
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="mBitRate">
     <property name="currentText">
      <string>低</string>
     </property>
     <item>
      <property name="text">
       <string>低</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>高</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>质量:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="mScreenShotTemplate"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>输出目录:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mOutputDir">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mVideoTemplate"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>视频模板名称:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>捕获源:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mCaptureSource">
     <item>
      <property name="text">
       <string>主窗口</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>当前窗口</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mCaptureSource</tabstop>
  <tabstop>mOutputDir</tabstop>
  <tabstop>mChooseDir</tabstop>
  <tabstop>mScreenShotTemplate</tabstop>
  <tabstop>mVideoTemplate</tabstop>
  <tabstop>mBitRate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
